modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceLocalServiceImpl.java

```diff
@@ -259,6 +259,23 @@ public void deleteSegmentsExperiences(
 		}
 	}
 
+	@Override
```

Comment:
As this method just call the addSegmentsExperience, what do you think about not create this method until it is needed?

Reply:
I buy your idea. Let's apply the YAGNI principal. Thanks @cgoncas